WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -23036,6 +23036,7 @@
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OBJC_BRIDGING_HEADER = "WordPressStatsWidgets/Supporting Files/WordPressStatsWidgets-Bridging-Header.h";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;
```

Comment:
Sad it had to come to this in c7ffd640a7d388559789802e2fa25ba75e674691.

Hopefully we can remember to check this when a new beta comes.

Actually... What do you think about tracking known workaround like this one in a checklist somewhere, so that we can run it when a new beta is release?

Reply:
Sounds like a good idea. Any suggestions where to track this kind of issue?

Reply:
Would be helpful to know what made us unable to turn that back on.

Like, for the case where Swift Bridging Header generated by Xcode contained warnings (due to some CoreData methods, iirm), we ended up finding a solution by trampolining to a header that added push/pop diagnostics pragma around it, and that worked great. So maybe for whatever case we had there that once again prevented us from turning it on there's also a way?

Reply:
I had looked for some Swift compiler flags to suppress this particular warning, but I couldn't find any. I don't think Swift compiler support diagnosis pragma directive in clang (or has anything similar to it).

But I just noticed the warnings has gone away in Xcode 14. I've opened https://github.com/wordpress-mobile/WordPress-iOS/pull/19310 to revert this change.